integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/search/LuceneNoLimitSearchIT.java

```diff
@@ -91,7 +125,7 @@ private static void initData() {
 		final IndexFieldReference<String> field;
 
 		IndexBinding(IndexSchemaElement root) {
-			field = root.field( "field", c -> c.asString() ).toReference();
+			field = root.field( "field", c -> c.asString().sortable( Sortable.YES ) ).toReference();
```

Comment:
I'm a bit worried now: was that necessary? It shouldn't be...

Reply:
it is not... I'm removing.